framework/src/bcs/ADIntegratedBC.C

```diff
@@ -99,14 +99,12 @@ ADIntegratedBCTempl<T>::computeResidual()
   for (auto & r : _residuals)
     r = 0;
 
-  if (_use_displaced_mesh)
-    for (_qp = 0; _qp < _qrule->n_points(); _qp++)
-      for (_i = 0; _i < _test.size(); _i++)
-        _residuals[_i] += raw_value(_ad_JxW[_qp] * _ad_coord[_qp] * computeQpResidual());
-  else
-    for (_qp = 0; _qp < _qrule->n_points(); _qp++)
-      for (_i = 0; _i < _test.size(); _i++)
-        _residuals[_i] += raw_value(_JxW[_qp] * _coord[_qp] * computeQpResidual());
```

Comment:
This is funny. First we specifically distinguish between needing AD or not and then we raw_value both cases anyways...

Reply:
I'm glad my silly code gives you giggles. Probably at some point I considered that only the `_ad_JxW` reference might be valid or properly computed in `Assembly` when using AD on the displaced mesh

Reply:
Or perhaps this is a remnant of the days when AD kernels were templated on the `ComputeType`